### ISSUE TITLE
Option to remove ads

### ISSUE THREAD

**Issue by Rosuav** [AnthorNet/SC-InteractiveMap#125]
The ads are starting to get obnoxious, and I'd really like to remove them. Quite a number of sites have an option to pay (often a one-time fee) to remove ads from the site. Would you consider adding something like that? I'd definitely be willing to support your work, given how much I use this while I'm playing.

**Comment by AnthorNet** [comment]
Hey,
I'm really sorry about that... I know it can be a pain but the site being my full time job now, I'm pretty sure you can understand ;)
That said, yep I do offer a month of ad free experience for every donation made on Paypal, or for all patreon, no matter the amount ;) you can find the links on the homepage.

**Comment by Rosuav** [comment]
No no, that's exactly what I was asking for :) So since that's the case, let me rephrase the request:

Can you make it clearer that donating removes ads? Because I looked around and didn't see that. In fact, I specifically browsed your Patreon reward tiers to try to find out if one of them included ad removal. 

I absolutely DO understand your need for revenue, but I also believe that it's in my interests to support you directly rather than get ads :)

**Comment by AnthorNet** [comment]
Thanks a lot for your support!
I'll try to make that a bit more clear.

You know dev aren't good marketers :D

**Comment by Rosuav** [comment]
Am dev myself. Know this all too well. :D
